core/src/tangram.cpp

```diff
@@ -887,6 +887,18 @@ void Map::runAsyncTask(std::function<void()> _task) {
     }
 }
 
+void Map::onMemoryWarning() {
+    auto& tileCache = impl->tileManager.getTileCache();
+
+    if (tileCache) {
+        tileCache->clear();
+    }
+
+    if (impl->scene && impl->scene->fontContext()) {
+        impl->scene->fontContext()->releaseFonts();
```

Comment:
I need to check this but probably you already did, when/how will the fontContext be loaded again post memory warning cleanup? As I see it requires a scene init again, is that how a client app will handle this post memory warning opps are done?